google/ggrc-core
# GGRC-6626/GGRC-6688 [PART 1] Replace mappings-tree-view with cycle-task-objects

# Issue description

In the scope of first part, `mappings-tree-view` component should be refactored and cleaned up.

# Steps to test the changes

1. Open info pane for any Cycle Task.
2. Map objects to CT with different types.

Actual result: Mapped objects are shown. Grey triangle is shown. 

Expected result:
1. Mapped objects should be showed in the CT's list.
2. Mapped objects with long titles should be truncated via `...` symbols.
3. Open-close functionality should be removed (gray triangle). 
4. List with mapped objects should be updated if objects are mapped/unmapped via CT Edit modal.

# Solution description

`mappings-tree-view` component is used only for сycle tasks. So, it was moved to appropriate template for cycle tasks. `mappings-tree-view` was renamed to `cycle-task-objects`.

# Sanity checklist




# PR Review checklist
